topics/tailwind/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss
```

Comment:
You could also add tailwind-scaffolding because it's in use: https://github.com/topics/tailwind-scaffolding.